chapters.md

```diff
@@ -60,13 +113,81 @@ Track/TrackTranslate               |   -   |  X
 
-See [the section on the `Linked Segments`](#linked-segments)) for more information about `Hard Linking`, `Soft Linking` and `Medium Linking`.
+See [the section on the `Linked Segments`](notes.md/#linked-segments) for more information about `Hard Linking`, `Soft Linking` and `Medium Linking`.
+
+## ChapterAtom
+The `ChapterAtom` is also called a `Chapter`.
+A `Chapter` element can be used recursively. Such a child `Chapter` is called `Nested Chapter`. 
+
+### ChapterUID
+
+The `ChapterUID` is a mandatory Matroska element with a unique ID with a range from 1 to 18446744073709551615.
+
+### ChapterStringUID
+A unique string ID to identify the `Chapter`.  
+Use for [WebVTT cue identifier storage](http://dev.w3.org/html5/webvtt/#webvtt-cue-identifier).
+
+### ChapterTimeStart
+A not scaled timestamp of the start of `Chapter` with nanosecond accuracy.  
+For `Simple Chapters` are the start time stamps equal to the chapter markers in the timeline.
+
+### ChapterTimeEnd
+A timestamp of the end of `Chapter` with nanosecond accuracy.  The timestamp is excluded and also not scaled.
+The end timestamp is used when the `Edition EditionFlagOrdered Flag` is set to `true`.
+A `Matroska Player` have to calculate a duration for this `Chapter` with the difference of end timestamp and start timestamp.  
+The end timestamp MUST be greater than the start timestamp otherwise the duration would be negative which is illegal. If the duration of a `Chapter` is 0, this `Chapter` will be ignored by a `Matroska Player`.
+
+Chapter   | Start timestamp | End timestamp | Duration
+----------|-----------------|---------------|------
+Chapter 1 | 0               | 1000000000    | 1000000000
+Chapter 2 | 1000000000      | 5000000000    | 4000000000
+Chapter 3 | 6000000000      | 6000000000    | 0 (chapter not used)
+Chapter 4 | 9000000000      | 8000000000    | -1000000000 (illegal)
+
+In the Matroska menu systems (Native,DVD) is the usage of an end timestamp depended on the current process. 
+
+### ChapterFlagHidden
+
+The `ChapterFlagHidden Flag` works a bit different as the `EditionFlagHidden Flag`. Each `Chapters ChapterFlagHidden Flag` works independent.  
+A `Nested Chapter` with `ChapterFlagHidden Flag` set to false remains visible even if the `Parent Chapter ChapterFlagHidden Flag` is set to true.
+
+Chapter + Nested Chapter | ChapterFlagHidden | visible
+-------------------------|-------------------|--------
+Chapter 1                | false             | yes
++Nested Chapter 1.1      | false             | yes
++Nested Chapter 1.2      | true              | no
+Chapter 2                | true              | no
++Nested Chapter 2.1      | false             | yes
++Nested Chapter 2.2      | true              | no
+
+### ChapterFlagEnabled
+
+If the `ChapterFlagEnabled Flag` is set to `true` a `Matroska Player` MUST use this `Chapter`.  
```

Comment:
Using chapters is not required for players. Web browsers don't use them and the files still play.